src/bankid_sdk/contrib/django/request.py

```diff
@@ -24,19 +25,27 @@ def strict_constant(obj: str) -> NoReturn:
     raise ValueError(f"Out of range float values are not JSON compliant: {obj!r}")
 
 
-def parse_json_body(
-    view: Callable[[HttpRequest, dict[str, Any]], HttpResponse], /
-) -> Callable[[HttpRequest], HttpResponse]:
-    @wraps(view)
-    def inner(request: HttpRequest) -> HttpResponse:
-        meta = request.META
-        content_type = meta.get("CONTENT_TYPE", meta.get("HTTP_CONTENT_TYPE")) or ""
-        if not content_type.startswith("application/json"):
-            return JsonResponse(
-                {"detail": f"Unsupported media type {content_type!r} in request"},
-                status=HTTPStatus.UNSUPPORTED_MEDIA_TYPE,
-            )
+class ParseRequestError(Exception):
+    def __init__(self, *, detail: str | None = None, status: int | None = None) -> None:
```

Comment:
I think we can have required arguments? And status should be HTTPStatus here I think

```suggestion
    def __init__(self, *, detail: str, status: HTTPStatus) -> None:
```